### ISSUE TITLE
[LWC] Cell editor does not work in Firefox

### ISSUE THREAD

**Issue by bmblb** [bryntum/support#7849]
Reproducible on https://bryntum-dev-ed.develop.lightning.force.com/lightning/n/BryntumGrid with following conditions:
1. LWS in on
2. Debug mode is off

**Comment by chuckn0rris** [comment]
This is known Firefox bug with Promises https://bugzilla.mozilla.org/show_bug.cgi?id=1663090
Which also mentioned in Salesforce docs here https://developer.salesforce.com/docs/platform/lwc/guide/security-lwsec-async.html?_ga=2.249124807.1359724086.1730723767-755359336.1698399851
with a workaround mentioned 

> You can get around this issue in Firefox by pre-compiling the code that’s in the static resource so the async function is replaced by equivalent code.

**Comment by chuckn0rris** [comment]
Another report is here https://trailhead.salesforce.com/trailblazer-community/feed/0D54V00007YvoqdSAB

**Comment by SergeyMaltsev** [comment]
TL;DR;

We use rollup with babel to compile LWC UMD bundle.

Babel’s @babel/preset-env determines which plugins to include by analyzing the target environments specified in the configuration and comparing them with the ECMAScript features supported in each environment. The preset uses a database of feature support tables (sourced from [compat-table](https://github.com/mdn/browser-compat-data) and [browserslist](https://github.com/browserslist/browserslist)) to decide if a specific transformation is needed for the target environments.

The minimum compatible chrome version is `54` which includes `transform-async-to-generator` plugin is  `{ chrome < 55 }`

For `chrome : 54` the list is:

```
Using plugins:
  transform-unicode-sets-regex { chrome < 112 }
  proposal-class-static-block { chrome < 94 }
  proposal-private-property-in-object { chrome < 91 }
  proposal-class-properties { chrome < 74 }
  proposal-private-methods { chrome < 84 }
  proposal-numeric-separator { chrome < 75 }
  proposal-logical-assignment-operators { chrome < 85 }
  proposal-nullish-coalescing-operator { chrome < 80 }
  proposal-optional-chaining { chrome < 91 }
  proposal-json-strings { chrome < 66 }
  proposal-optional-catch-binding { chrome < 66 }
  proposal-async-generator-functions { chrome < 63 }
  proposal-object-rest-spread { chrome < 60 }
  transform-dotall-regex { chrome < 62 }
  proposal-unicode-property-regex { chrome < 64 }
  transform-named-capturing-groups-regex { chrome < 64 }
  transform-async-to-generator { chrome < 55 }      <--------------------------------------------------------
  proposal-export-namespace-from { chrome < 72 }
  syntax-dynamic-import
  syntax-top-level-await
  syntax-import-meta
```


For the most common  `browsers : 'last 2 versions'` config  includes:

```
Using targets:
{
  "android": "128",
  "chrome": "128",
  "edge": "127",
  "firefox": "129",
  "ie": "10",     <--------------------------------------------------------
  "ios": "17.6",
  "opera": "110",
  "opera_mobile": "80",
  "safari": "17.6",
  "samsung": "24"
}
```

The list is much bigger which we don't want

```
Using plugins:
  transform-unicode-sets-regex { ie, samsung }
  proposal-class-static-block { ie }
  proposal-private-property-in-object { ie }
  proposal-class-properties { ie }
  proposal-private-methods { ie }
  proposal-numeric-separator { ie }
  proposal-logical-assignment-operators { ie }
  proposal-nullish-coalescing-operator { ie }
  proposal-optional-chaining { ie }
  proposal-json-strings { ie }
  proposal-optional-catch-binding { ie }
  transform-parameters { ie }
  proposal-async-generator-functions { ie }
  proposal-object-rest-spread { ie }
  transform-dotall-regex { ie }
  proposal-unicode-property-regex { ie }
  transform-named-capturing-groups-regex { ie }
  transform-async-to-generator { ie }
  transform-exponentiation-operator { ie }
  transform-template-literals { ie }
  transform-literals { ie }
  transform-function-name { ie }
  transform-arrow-functions { ie }
  transform-block-scoped-functions { ie < 11 }
  transform-classes { ie }
  transform-object-super { ie }
  transform-shorthand-properties { ie }
  transform-duplicate-keys { ie }
  transform-computed-properties { ie }
  transform-for-of { ie }
  transform-sticky-regex { ie }
  transform-unicode-escapes { ie }
  transform-unicode-regex { ie }
  transform-spread { ie }
  transform-destructuring { ie }
  transform-block-scoping { ie }
  transform-typeof-symbol { ie }
  transform-new-target { ie }
  transform-regenerator { ie }
  proposal-export-namespace-from { ie }
  syntax-dynamic-import
  syntax-top-level-await
  syntax-import-meta
```
